datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java

```diff
@@ -80,13 +82,13 @@ public class SdxRuntimeUpgradeService {
     @Inject
     private SdxUpgradeFilter upgradeFilter;
 
-    public SdxUpgradeResponse checkForUpgradeByName(String clusterName, SdxUpgradeRequest upgradeSdxClusterRequest, String accountId) {
```

Comment:
Is there any reason why its called `prepare` here and `upgradePreparation` elsewhere?

Reply:
+1 on this

Reply:
if it bothers you I can rename